qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -58,10 +58,10 @@ class Grover(AmplitudeAmplifier):
     The details of how :math:`S_f` works are unimportant to the algorithm; Grover's
     search algorithm treats the oracle as a black box.
 
-    This class supports oracles in form of :class:`~qiskit.QuantumCircuit`
+    This class supports oracles in form of a :class:`~qiskit.QuantumCircuit`.
```

Comment:
```suggestion
    This class supports oracles in form of a :class:`~qiskit.circuit.QuantumCircuit`.
```
Adding Matthews suggested change which I could not apply since it said it was outdated